game-core/src/main/java/games/strategy/engine/data/properties/ComboProperty.java

```diff
@@ -31,7 +32,7 @@ public ComboProperty(final String name,
       final T defaultValue,
       final Collection<T> possibleValues) {
     super(name, description);
-    this.possibleValues = new ArrayList<>(possibleValues);
+    this.possibleValues = Collections.unmodifiableCollection(new ArrayList<>(possibleValues));
```

Comment:
In this case ImmutableList would've been a shorter version, but I guess that works too

Reply:
Please see this comment in the PR description:

> I considered changing the type of `possibleValues` to Guava's `ImmutableCollection`.  However, the enclosing type is serializable, and since we've experienced issues in the past serializing the Guava immutable collection types, I didn't want to risk introducing a regression.